toolbox/githubctl/main.go

```diff
@@ -245,17 +245,23 @@ func CreateIstioReleaseUploadArtifacts() error {
 // DailyReleaseQualification triggers test jobs buy creating a PR that generates
 // a GitHub notification. It blocks until PR status is known and returns nonzero
 // value if failure. Links to test logs will also be logged to console.
-func DailyReleaseQualification() error {
+func DailyReleaseQualification(baseBranch *string) error {
```

Comment:
there are other places in the code where baseBranch is used, I did not want to go through them. Say What, lets look at it tomorrow.

Reply:
sure but they are not in conflict with each other since only one of the functions will be called, which is selected using `--op=` flag. Flags like `--base_branch` is just a way for users to pass parameters through command line.